pythonsd/urls.py

```diff
@@ -1,20 +1,20 @@
-from django.conf.urls import url
+from django.urls import path
 from django.views import generic
 
 from .views import MeetupWidget
 
 
 urlpatterns = [
-    url(
-        r"^$",
+    path(
+        "",
         generic.RedirectView.as_view(url="/index.html", permanent=False),
         name="home",
     ),
-    url(
-        r"^meetup-widget\.(?P<format>html|json)$",
+    path(
+        "meetup-widget.<str:format>",
         MeetupWidget.as_view(),
         name="meetup-widget",
     ),
-    url(r"^coc/?$", generic.RedirectView.as_view(url="/pages/code-of-conduct.html")),
+    path("coc", generic.RedirectView.as_view(url="/pages/code-of-conduct.html")),
```

Comment:
This works on `/coc` but not `/coc/`. I feel like we should fix that (and maybe have a test).

Reply:
Added a test and a new route.  I felt that was the simplest answer.